components/forms/w-affix.vue

```diff
@@ -1,8 +1,10 @@
 <script setup>
-import { suffix, prefix } from "@warp-ds/css/component-classes";
-import { computed } from "vue";
-import IconClose16 from '@warp-ds/icons/vue/close-16'
-import IconSearch16 from "@warp-ds/icons/vue/search-16";
+import { computed } from 'vue';
+
+// eslint-disable-next-line vue/no-dupe-keys
+import { suffix, prefix } from '@warp-ds/css/component-classes';
```

Comment:
Maybe we should change to `import { suffix as ccSuffix, prefix as ccPrefix } from '@warp-ds/css/component-classes';` here, and `const classBase = computed(() => (props.prefix ? ccPrefix : ccSuffix));` on line 17?